doc-content/enterprise-only/openshift/customimage-rpm-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='customimage-rpm-proc_{context}']
+:componentimage: kieserver
+:component: {KIE_SERVER}
+= Creating a custom {KIE_SERVER} image with an additional RPM package
+
+You can create a custom {KIE_SERVER} image where an additional RPM package is installed. You can push this image into your custom registry and then use it to deploy the {KIE_SERVER}.
+
+You can install any package from the Red Hat Enterprise Linux repository. This example installs the `procps-ng` package, which provides the `ps` utility, but you can modify it to install other packages. 
+
+.Procedure
+
+include::customimage-download-step.adoc[]
+. Create a `Dockerfile` that defines a custom image based on the base image. The file must chnge the current user to `root`, install the RPM package using the `yum` command, and then revert to `USER 185`, the {EAP} user. The following example shows the content of the `Dockerfile` file:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+FROM registry.redhat.io/{PRODUCT_INIT}-7/{PRODUCT_INIT}-kieserver-rhel8:7.7.0
+USER root
+RUN yum -y install procps-ng
+USER 185
+----
++
+Replace the name of the RPM file as necessary. You might need to install several RPM files, in this case, use several `RUN` commands.
```

Comment:
If you wanted to instead use an example of installing an rpm that has other rpm dependencies, you could use httpd (instead of procps-ng). I was able to verify that this works, just by replacing "procps-ng" with "httpd" in the Dockerfile. I verified that /usr/sbin/httpd was installed, and all the dependencies were automatically installed as well.

However, you might want to continue just with procps-ng, since the running the ps command is an easy way to show in the docs that it got installed properly. Really I just wanted to let you know that dependencies will get installed automatically as well, if you wanted to mention that here. Might be good to so users don't think they have to install all the rpm dependencies individually.